core/src/main/scala/replpp/util/package.scala

```diff
@@ -7,7 +7,33 @@ import scala.jdk.CollectionConverters.*
 import scala.util.{Try, Using}
 
 package object util {
-  
+
+  /** `to` is included in this Range */
+  case class Range(from: Int, to: Int)
```

Comment:
the `Range` from the Scala standard library is exclusive, so maybe rename this to something a bit more obviously different, e.g. `IRange`? Could also consider just using `scala.collection.immutable.Range.Inclusive`, although that also has a (for us) superfluous `step`.

Reply:
yeah i actually considered Range but the `step` made me decide against it. Fair point about the different default re inclusive/exclusive - I've just renamed this range to `InclusiveRange`